ansible/roles/test/vars/testcases.yml

```diff
@@ -259,6 +259,12 @@ testcases:
       required_vars:
           ptf_host:
 
+    warm-reboot-vnet:
```

Comment:
the list of topologies is aligned with vnet test but not with the warm-reboot test. need to check what is the right topology list